modeshape-parent/pom.xml

```diff
@@ -396,80 +338,17 @@
                     <value>postgresql9</value>
                 </property>
             </activation>
-            <dependencies>
-                <dependency>
-                    <!--Must be JDBC 4 (Java 6) compliant-->
-                    <groupId>postgresql</groupId>
-                    <artifactId>postgresql</artifactId>
-                    <version>${version.postgresql.9}</version>
-                    <scope>test</scope>
-                </dependency>
-            </dependencies>
             <properties>
                 <database>postgresq9</database>
-                <dataSource.driverClassName>org.postgresql.Driver</dataSource.driverClassName>
-                <dataSource.url>jdbc:postgresql://db14.mw.lab.eng.bos.redhat.com:5432/dballo04</dataSource.url>
+                <database.name>dballo04</database.name>
```

Comment:
since we're using docker urls, these 3 properties should be changed from `dballo` to `postgres` really